dash_core_components/version.py

```diff
@@ -1 +1 @@
-__version__ = '0.40.0'
+__version__ = '0.40.2'
```

Comment:
This version is inconsistent with the others!

Reply:
Fixed typo :)